lua/nvim-treesitter/shell_command_selectors.lua

```diff
@@ -151,22 +155,40 @@ function M.select_download_commands(repo, project_name, cache_folder, revision)
       M.select_install_rm_cmd(cache_folder, project_name..'-tmp')
     }
   else
+    local git_folder = utils.join_path(cache_folder, project_name)
+    local clone_error = 'Error during download, please verify your internet connection'
+    if is_windows then
+      clone_error = clone_error .. ". If on Windows you may need to enable Developer mode"
```

Comment:
I would add a link or not mention it. As a user I would have no idea what developer mode is and how I can enable it.

Reply:
Yeah I agree on that point too, maybe you could even suggest to open an issue.